src/sentry/integrations/vercel/client.py

```diff
@@ -39,8 +43,22 @@ def get_user(self):
         return self.get_cached(self.USER_URL)["user"]
 
     def get_projects(self):
-        # TODO: we will need pagination since we are limited to 20
-        return self.get(self.PROJECTS_URL)["projects"]
+        limit = 20
+        params = {"limit": limit}
+        projects = []
+        # no one should have more than 200 projects
```

Comment:
This probably won't age well -- do the projects get cached? can we filter down projects as the user types?

Reply:
@billyvg No, we don't' cache the projects. Theoretically, we could filter down as the user types but that would but a much bigger change since we don't support search like that. If this does become a problem and we think that search is the right solution, we can look into that. But I rather not spend time optimizing this before we think it's needed.